metacat-main/src/main/java/com/netflix/metacat/main/api/v1/TagController.java

```diff
@@ -559,12 +555,10 @@ private void removeResourceTags(final TagRemoveRequestDto tagRemoveRequestDto) {
         switch (name.getType()) {
             case CATALOG:
                 //catalog service will throw exception if not found
-                final CatalogDto catalogDto = this.catalogService.get(name, GetCatalogServiceParameters.builder()
+                this.catalogService.get(name, GetCatalogServiceParameters.builder()
                     .includeDatabaseNames(false).includeUserMetadata(false).build());
-                if (catalogDto != null) {
-                    this.tagService.removeTags(name, tagRemoveRequestDto.getDeleteAll(),
```

Comment:
The catalogDto will always NOT be null or the tag API can handle null internally?